pkg/logging/redirectstderr_linux.go

```diff
@@ -1,7 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache v2.0 license.
 
-//+build unix
+//go:build unix
```

Comment:
this and below `_windows.go` changes were automatically updated from `make all`

Reply:
This is coming only from 1.17.1?

Reply:
I don't think so, I tried same command with go 1.19.2 version and it generated more of this kind of formatting changes

Reply:
typically we include these files. But when I included `redirectstderr_*.go` files, they broke my builds. But let's keep them for now and see if the builds fail for this PR. If they fail, let's fix them.